common/include/villas/kernel/devices/generic_driver.hpp

```diff
@@ -0,0 +1,52 @@
+/* GenericDriver - Works for standard Linux drivers
```

Comment:
```suggestion
/* GenericDriver - Works for standard Linux drivers.
```

Reply:
This is not descriptive at all. Please briefly summarize what a generic driver is.

Does it abstract a Linux Kernel device? FPGA Internal IP component? VFIO abstraction?